proxy/fadein.go

```diff
@@ -5,19 +5,19 @@ import (
 	"math/rand"
 	"time"
 
-	"github.com/zalando/skipper/eskip"
 	"github.com/zalando/skipper/routing"
 )
 
-func returnFast(rt *routing.Route) bool {
-	if rt.BackendType != eskip.LBBackend {
-		return true
-	}
+type fadeIn struct {
+	rnd              *rand.Rand
+	endpointRegistry *routing.EndpointRegistry
+}
 
+func (f *fadeIn) noFadeIn(rt *routing.Route) bool {
```

Comment:
Do we need this? Maybe its better to simply inline the condition

Reply:
It was intended to increase readability (literally tell the people reading this code "this is the case when there is no fade-in"), however, I agree it may be too obvious so I decided to inline it.